CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.17.4-dev
+* Bug - Fixed a bug where Docker Version() API never returns by adding a timeout [#1363](https://github.com/aws/amazon-ecs-agent/pull/1363)
```

Comment:
Should we put Version() in `` ?

Reply:
I don't think it matters. I can modify it if you really insist (prefer not to 😃 ).